### ISSUE TITLE
Vulnerability in Dovecot

### ISSUE THREAD

**Issue by Nutomic** [hardware/mailserver#370]
#### Classification
* [x] Vulnerability

There was a vulnerability discovered in Dovecot yesterday. You should update the image to release the fix. Details:

https://www.debian.org/security/2019/dsa-4418

I'm not sure how often new Docker builds are triggered, but it might be a good idea to do automatic, daily builds to cover cases like this one.

**Comment by denji** [comment]
https://github.com/hardware/mailserver/blob/master/docker-compose.sample.yml#L52-L57

Fastfix: Close this port

**Comment by denji** [comment]
https://seclists.org/fulldisclosure/2019/Apr/34

**Comment by navossoc** [comment]
I think this issue is already fixed now.

> For the stable distribution (stretch), this problem has been fixed in version 1:2.2.27-3+deb9u4.

```
# dovecot --version
2.2.27 (c0f36b0)
```

```
# apt list --installed | grep dovecot

WARNING: apt does not have a stable CLI interface. Use with caution in scripts.

dovecot-core/now 1:2.2.27-3+deb9u4 amd64 [installed,local]
dovecot-imapd/now 1:2.2.27-3+deb9u4 amd64 [installed,local]
dovecot-lmtpd/now 1:2.2.27-3+deb9u4 amd64 [installed,local]
dovecot-managesieved/now 1:2.2.27-3+deb9u4 amd64 [installed,local]
dovecot-mysql/now 1:2.2.27-3+deb9u4 amd64 [installed,local]
dovecot-pgsql/now 1:2.2.27-3+deb9u4 amd64 [installed,local]
dovecot-pop3d/now 1:2.2.27-3+deb9u4 amd64 [installed,local]
dovecot-sieve/now 1:2.2.27-3+deb9u4 amd64 [installed,local]
```
